app/2.0/start/quick-tour.md

```diff
@@ -6,7 +6,8 @@ title: Try Polymer
 
 Polymer makes it simple to create web components, declaratively.
 
-New web developers can simply add custom HTML elements on a web page with markdown. It’s just like using the HTML tags you’re already familiar with:
+New web developers can simply add custom HTML elements on a web page with markdown.
```

Comment:
Eeek. Just noticed this: should be "markup" ... Or perhaps just, "on a web page." HTML is a markup language, Markdown is a plain-text format for authoring HTML...